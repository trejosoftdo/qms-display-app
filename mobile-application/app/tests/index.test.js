import React from 'react';
import Home from '../';


describe('Home route component', () => {
  it('renders the component as expected', () => {
    expect(renderShallow(<Home  />)).toMatchSnapshot();
  });
});
