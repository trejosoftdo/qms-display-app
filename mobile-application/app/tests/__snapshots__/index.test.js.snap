// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home route component renders the component as expected 1`] = `
<React.Fragment>
  <AppHeader
    headerRight={[Function]}
  />
  <TurnsStatusTableView />
</React.Fragment>
`;
